mkosi/resources/man/mkosi.1.md

```diff
@@ -326,9 +326,9 @@ grouped by section below.
 Configuration is parsed in the following order:
 
 * The command line arguments are parsed.
-* `mkosi.local.conf` or `mkosi.local` is parsed if it exists. This file or
-  directory should be in `.gitignore` (or equivalent) and is intended for local
-  configuration.
+* `mkosi.local.conf` and `mkosi.local/` are parsed if they exists (in that order).
+  This file or directory should be in `.gitignore` (or equivalent)
+  and is intended for local configuration.
```

Comment:
Sorry, late to the game, the "this file or directory" makes sense logically, but should probably be an "and", since above we now say that both are parsed. In the last line it should it should also read "are" instead of "is", since both are intended for local configuration.

Reply:
-> 3581